pom.xml

```diff
@@ -82,9 +82,9 @@
         <arquillian.version>1.1.7.Final</arquillian.version>
         <cdi.tck.version>1.2.8.Final</cdi.tck.version>
         <httpclient.version>4.5.2</httpclient.version>
-        <osgi.servlet.range>[2.4,)</osgi.servlet.range>
-        <osgi.el.range>[2.2,)</osgi.el.range>
-        <osgi.faces.range>[2.0,)</osgi.faces.range>
+        <osgi.servlet.range>2.4</osgi.servlet.range>
```

Comment:
we run with servlet 2.5, 3.0, 3.1 and 4.x so we need a range there I fear, why removing ranges instead of making them "valid"?